assets/js/components/EntitySearchInput.js

```diff
@@ -137,8 +137,9 @@ function EntitySearchInput() {
 				<div className="googlesitekit-entity-search__actions">
 					<Button
 						onClick={ onClose }
-						trailingIcon={ <CloseDark width="30" height="20" /> }
+						trailingIcon={ <CloseDark width="20" height="20" /> }
 						className="googlesitekit-entity-search__close"
+						title="Close"
```

Comment:
@hussain-t Strings must be localized.
```suggestion
						title={ __( 'Close', 'google-site-kit' ) }
```